src/Analytics/TonesOverTime.js

```diff
@@ -228,31 +237,38 @@ const TonesOverTime = () => {
       },
     },
   };
-
+   if (noCharts === true ) {
+     return(
+        <h4>
+          You do not have any dream data, add dreams to see your dream
+          tones
+        </h4> 
+     )
```

Comment:
I like this, will be nice if we decide to add any sort of graphic or anything here later.

Reply:
Agree but for now this is a good place holder